core/src/main/java/nl/nn/adapterframework/http/HttpSenderBase.java

```diff
@@ -367,13 +373,17 @@ public void configure() throws ConfigurationException {
 			javax.net.ssl.SSLSocketFactory socketfactory = (javax.net.ssl.SSLSocketFactory) javax.net.ssl.SSLSocketFactory.getDefault();
 			sslSocketFactory = new SSLConnectionSocketFactory(socketfactory, hostnameVerifier);
```

Comment:
The alias password now only works when a keystoreUrl is provided, but what if you want to use the default keystore?